help/md/mcmcmc.md

```diff
@@ -9,7 +9,7 @@ The Mcmcmc analysis object produced by a call to this function keeps copies of t
 
 An MCMCMC analysis is initiated using the `mcmcmc.run()` method.  
 The `StoppingRule[]` argument provides a mechanism to automatically terminate a run once a set of rules are met: perhaps once the run has attained convergence, or after a certain amount of time has passed.  The run will be terminated once *all* convergence rules ([`srGelmanRubin()`], [`srGeweke()`], [`srMinESS()`], [`srStationarity()`]) have been fulfilled; or once *any* threshold rules ([`srMaxTime()`], [`srMaxIteration()`]) are met.
-The parameters `checkpointFile` and `checkpointInterval` generate snapshots of the current state of the MCMCMC run from which the run can be continued if interrupted using the `mcmc.initializeFromCheckpoint()` method.
+The parameters `checkpointFile` and `checkpointInterval` generate snapshots of the current state of the MCMCMC run from which the run can be continued if interrupted using the `mcmc.initializeFromCheckpoint()` method. More details are given on the documentation page for [`mcmc()`].
```

Comment:
I would maybe say "An example is" instead of "More details are". But I wouldn't object to just including a checkpointing example on this page too.